apps/fabric-website/src/pages/HomePage/HomePage.base.tsx

```diff
@@ -0,0 +1,264 @@
+        </div>
+      </div>
+    );
+  };
+
+  private _renderPlatformsSection = (): JSX.Element => {
+    const { isMountedOffset } = this.state;
+    const { theme, styles } = this.props;
+
+    // We need to get classNames within this method for offset transitions and inverted sections.
+    const classNames = getClassNames(styles, {
+      theme,
+      isMountedOffset,
+      isInverted: true
+    });
+
+    return (
+      <section className={classNames.platformsSection}>
+        <div className={classNames.sectionContent}>
+          <div className={classNames.oneHalf}>
+            <h2 className={classNames.platformsTitle}>Build across platforms</h2>
+          </div>
+          <div className={classNames.oneFourth}>
+            <p>
+              We're broadening our guidance to include more platforms and create an open source system, making it possible for us all to
+              evolve together.
+            </p>
+          </div>
+        </div>
+      </section>
+    );
+  };
+
+  private _renderResourcesSection = (): JSX.Element => {
+    return (
+      <section className={this._classNames.resourcesSection}>
+        <div className={this._classNames.sectionContent}>
+          <div className={this._classNames.oneHalf}>
+            <Image
+              src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/discover-resources-1x.png"
+              alt="Resources illustration"
+              className={this._classNames.illustration}
+            />
+          </div>
+          <div className={this._classNames.oneFourth}>
+            <h2 className={this._classNames.resourcesTitle}>Discover resources</h2>
+            <p>Find design, inclusive and developer onboarding resources, and learn about how to become a contributor.</p>
+            <p>{this._renderLink('#/resources', 'See resources')}</p>
+          </div>
+        </div>
+      </section>
+    );
+  };
+
+  private _renderUsageSection = (): JSX.Element => {
+    return (
+      <section className={this._classNames.usageSection}>
+        <div className={this._classNames.sectionContent}>
+          <div className={this._classNames.oneFourth}>
+            <h2 className={this._classNames.usageTitle}>Who at Microsoft uses Fabric?</h2>
+            <p>From Word, PowerPoint and Excel to PowerBI, many teams in Microsoft utilize the functionality of Fabric.</p>
+          </div>
+          <div className={this._classNames.oneFourth} />
+          <figure className={this._classNames.oneHalf}>
+            <ul className={this._classNames.usageIconList}>{this._renderUsageIconList()}</ul>
+            <figcaption>
+              <strong>+many additional Microsoft sites and products</strong>
+            </figcaption>
+          </figure>
+        </div>
+      </section>
+    );
+  };
+
+  /**Renders a link with an icon */
+  private _renderLink = (url: string, text: React.ReactNode, disabled?: boolean, isCTA?: boolean, icon = 'Forward'): JSX.Element => {
```

Comment:
What is CTA?

Reply:
"Call To Action", essentially the primary ask for a user on a page.